includes/tracks/class-wc-tracks.php

```diff
@@ -65,6 +77,16 @@ public static function get_identity( $user_id ) {
 
 	}
 
+	public static function update_revenue_cache() {
+		update_option( self::REVENUE_CACHE_OPTION, WC_Tracker::get_order_totals() );
+	}
+
+	public static function get_total_revenue() {
+		$total_revenue = get_option( self::REVENUE_CACHE_OPTION );
```

Comment:
We could solve that update think here, something like:

```php
$total_revenue = get_option( self::REVENUE_CACHE_OPTION, false ); // If don't exists defaults to false.

if ( false === $total_revenue ) {
	$total_revenue = WC_Tracker::get_order_totals();

	update_option( self::REVENUE_CACHE_OPTION, $total_revenue );
}

return empty( $total_revenue['gross'] ) ? null : $total_revenue['gross'];
```

Reply:
Done in 43fd21c.